node_modules/hubot-plusplus/src/plusplus.coffee

```diff
@@ -62,15 +62,11 @@ module.exports = (robot) ->
     # if we got a score, then display all the things and fire off events!
     if score?
       message = [
-        
-        ]
-        
-        
-      message = if reason?
-                  "#{name} has #{score} points, #{reasonScore} of which are for #{reason}"
-                else
-                  "#{name} has #{score} points"
-      message = "#{message}." if ['.', '?', '!'].indexOf(message.slice(-1)) == -1
+        "#{name} has #{score} ",
+        if score == 1 then 'point' else 'points',
+        if reason? then ", #{reasonScore} of which are for #{reason}", else ''
+        if ['.', '?', '!'].indexOf(reason.slice(-1)) == -1 then '.' else ''
```

Comment:
CoffeeScript has a splice operator and an `in` operator to check for array membership, so you could re-write this as

```coffee
if reason[-1..] not in ['.', '?', '!'] then '.' else ''
```

Also, reason might not exist and throw an error here.